app/models/maths_and_physics/eligibility_answers_presenter.rb

```diff
@@ -128,5 +128,13 @@ def degree_answer
       else I18n.t("maths_and_physics.answers.has_uk_maths_or_physics_degree.#{eligibility.has_uk_maths_or_physics_degree}")
       end
     end
+
+    def qts_answer_academic_year
+      if eligibility.awarded_qualified_status_on_or_after_cut_off_date?
+        MathsAndPhysics.first_eligible_qts_award_year
+      else
+        MathsAndPhysics.first_eligible_qts_award_year - 1
```

Comment:
In theory a user shouldn't be able to see this answer on the check-your-answers page, they should see the "ineligible" page first. Is there a reason why you decided to include it?

Reply:
It's funny, I originally had it assuming the answer was the eligible one, and have [done the same for the admin presenter](https://github.com/DFE-Digital/dfe-teachers-payment-service/commit/83ab345ae22a80f1f8a1d1fb0c8dafd660833f05) in the other branch. There is a part of me that does feel a little uncomfortable making the assumption that this presenter will only be used to display eligible claims though...

Reply:
Yeah, I think you're right. It is possible in an edge-case that a user may be able to see this answer on the check your answers page.